sopel_remind/plugin.py

```diff
@@ -37,6 +39,48 @@ def configure(settings: Config):
         default=settings.core.homedir)
     os.makedirs(settings.remind.location, exist_ok=True)
 
+    # manage migration while configuring the plugin
+    migrate: str = input(
+        'Do you want to migrate from the built-in remind plugin? Y/n'
+    ) or 'y'
+    migrate_attr = BooleanAttribute('mock_migrate', default=True)
+    if migrate_attr.parse(migrate):
+        # is there anything to migrate?
+        builtin_filename = os.path.join(
+            settings.core.homedir,
+            settings.basename + '.reminders.db',
+        )
+
+        migrated: int = 0
+        if os.path.isfile(builtin_filename):
+            filename = backend.get_reminder_filename(settings)
+            migrated = migrate_builtin(builtin_filename, filename)
+            os.remove(builtin_filename)
```

Comment:
```suggestion
            os.rename(builtin_filename, builtin_filename + '.migrated')
```

```suggestion
            os.rename(builtin_filename, builtin_filename + '.bak')
```

(or whatever other prefix/suffix makes more sense to you)